go.mod

```diff
@@ -1,134 +1,152 @@
 module github.com/onflow/rosetta
 
-go 1.19
+go 1.20
 
 require (
 	github.com/coinbase/rosetta-sdk-go v0.7.0
 	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.2.0
 	github.com/dgraph-io/badger/v2 v2.2007.4
 	github.com/dgraph-io/badger/v3 v3.2103.2
-	github.com/ethereum/go-ethereum v1.10.18
-	github.com/golang/protobuf v1.5.3
+	github.com/ethereum/go-ethereum v1.13.10
+	github.com/golang/protobuf v1.5.4
 	github.com/grpc-ecosystem/go-grpc-middleware v1.3.0
-	github.com/libp2p/go-libp2p v0.28.1
-	github.com/onflow/cadence v0.41.1
-	github.com/onflow/flow-go v0.32.1-0.20231005181129-9c6f9fff84a0
-	github.com/onflow/flow-go/crypto v0.24.9
-	github.com/onflow/flow/protobuf/go/flow v0.3.2-0.20230915140723-432828f7afb9
+	github.com/libp2p/go-libp2p v0.32.2
+	github.com/onflow/cadence v1.0.0-preview.22
+	github.com/onflow/crypto v0.25.1
+	github.com/onflow/flow-go v0.33.2-0.20240425000917-56461308db55 //v0.32.1-0.20231005181129-9c6f9fff84a0
```

Comment:
this isn't running against the `v0.33` branch, but actually a commit from master.

Can you try with `v0.33.17`